docs/source/markdown/podman-auto-update.1.md

```diff
@@ -70,6 +61,14 @@ For a container to send the READY message via SDNOTIFY it must be created with t
 | .Policy         | Auto-update policy of the container    |
 | .Updated        | Update status: true,false,failed       |
 
+#### **--rollback**
```

Comment:
We should get this change merged independently from this PR, this is obviously correct and should not have to wait for the other stuff to land which will take more time.

Reply:
I think this must be under the --quiet option.